cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -408,7 +410,7 @@ data:
       buffer_queue_full_action block
       # Set the chunk limit conservatively to avoid exceeding the recommended
       # chunk size of 5MB per write request.
-      buffer_chunk_limit 1M
+      buffer_chunk_limit 512k
       # Cap the combined memory usage of this buffer and the one below to
       # 1MiB/chunk * (6 + 2) chunks = 8 MiB
```

Comment:
Adjust the comment? It should be
```
512KiB/chunk * (6 + 2) chunks = 84 MiB
```
now, right?

Reply:
Updated